src/frontend/komponenter/Oppgavebenk/OppgaveList.tsx

```diff
@@ -120,7 +122,15 @@ const OppgaveList: React.FunctionComponent = () => {
                                             }
                                         </td>
                                         <td className={'beskrivelse'}>{oppg.beskrivelse}</td>
-                                        <td>{oppg.aktoerId}</td>
+                                        <td>
+                                            {
+                                                oppg.identer.find(
+                                                    (ident: IOppgaveIdent) =>
+                                                        ident.gruppe ===
+                                                        IdentGruppe.FOLKEREGISTERIDENT
+                                                )?.ident
```

Comment:
Hva skjer om man ikke finner noe her? Kanskje dette fungerer?

```suggestion
                                                oppg.identer.find(
                                                    (ident: IOppgaveIdent) =>
                                                        ident.gruppe ===
                                                        IdentGruppe.FOLKEREGISTERIDENT
                                                )?.ident ?: "Ukjent"
```